app/main.py

```diff
@@ -1,3 +1,64 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(self, Distance) and isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, int) or isinstance(other, float):
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: Distance | int | float) -> Distance:
+        if isinstance(self, Distance) and isinstance(other, Distance):
+            self.km += other.km
+            return self
+        elif isinstance(other, int) or isinstance(other, float):
+            self.km += other
+            return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        if isinstance(other, int) or isinstance(other, float):
+            return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> Distance:
+        if isinstance(other, int) or isinstance(other, float):
+            return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: Distance | int | float) -> bool:
+        if isinstance(self, Distance) and isinstance(other, Distance):
+            return True if self.km < other.km else False
```

Comment:
```suggestion
            return self.km < other.km
```
fix it in other cases

Reply:
> fix it in other cases

Fixed) And could you check my solution of decorators?)

Reply:
check guidelines on how to do a re-request review.